docs-src/docs/annotation-integration/introduction.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Introduction
+---
+
+In order to enable a programmatic interface to the ground truth production process, an HTTP API is made available. This 
+page aims at describing how to work with these integrations, what the available possibilities are, and the concepts and 
+identifiers used.
+
+The current possibilities are limited to reviewing delivered annotations.
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+* An account with permissions to use our API
+* Generated API credentials. See [API Credentials](../kognic_apis.md/#generating-credentials).
+* Installed our Python 3 SDK for authentication - [kognic-auth](https://pypi.org/project/kognic-auth/)
+
+## No API Client Available
+
+At the moment we do not provide an API client for this integration. Instead, we'll provide examples of how you can 
+interact with our API. 
+
+This page will also assume that the user has access to at least one project where at least one input batch has been 
+uploaded. (See the client docs regarding these concepts [here](../kognic-io/project).)
+
+## Endpoints
+You can discover the list of accessible endpoints within our 
+[swagger documentation](https://shape-integration.app.kognic.com/api).
+
+## Request Example
+Here's an example using the kognic-auth library to fetch the [feedback error types](review#feedback) available to the user:
+
+```python
+import requests
+
+from kognic.auth.requests.auth_session import RequestsAuthSession
+
+base_url = "https://annotation-integration.app.kognic.com/v1/"
+client = RequestsAuthSession()
+
+try:
+    response = client.session.get(base_url + "reviews/error-types")
+    response.raise_for_status()
+    data = response.json()
+    print(data)
+except requests.exceptions.RequestException as e:
+    print(f"Request error: {e}")
```

Comment:
Another way of doing it would be to leverage the `HttpClient`. 

```python
from pprint import pprint
from kognic.base_clients.http_client import HttpClient
from kognic.auth import DEFAULT_HOST

kognic_http_client = HttpClient(
    host="https://annotation-integration.app.kognic.com/v1/", auth_host=DEFAULT_HOST
)

pprint(kognic_http_client.get("reviews/error-types"))
```

What do you think is easiest? Ideally you shouldn't have to provide `DEFAULT_HOST`, seems like a bug we have to.

Reply:
Good idea. Right now I'm not confident that the http client lib provides a good user experience, I'd need to use it a bit more, and successfully:)

Reply:
Got it, mostly a thin wrapper. 

it's usually the one we extend to create "dedicated clients" such [as this one](https://github.com/annotell/export-python/blob/1e2b4d2b651d7b8495594c9d4e7e02baafc80fce/packages/kognic-export-api-client/src/kognic/export_api_client/client.py#L14)

Reply:
Interesting, thought this was only used in kognic-io. I think most of https://github.com/annotell/annotell-apiclients-python are based on the BaseApiClient. Not sure how they differ though..

Reply:
I've copied this from the data quality page in these docs: https://developers.kognic.com/docs/dataset-exploration/introduction/ so at least it's consistent! :joy: